backend/config/settings.py

```diff
@@ -382,16 +389,10 @@
     # Will not be enabled in cloud environments
     DISABLE_AUTH = False
 
-# Remove once all Census data has been migrated
```

Comment:
These are being removed before all Census data has been migrated. Is this correct?

Reply:
These are left over from the old data distro work. We should be able to delete them now without trouble, but could also wait until we do the full data_distro cleanup.